src/core/style/parser.js

```diff
@@ -39,13 +39,23 @@ function parseStyleNamedExpr(styleSpec, node) {
     if (node.operator != ':') {
         throw new Error('Invalid syntax');
     }
+    if (node.left.name.length && node.left.name[0] == '@') {
+        node.left.name = '__cartovl_variable_' + node.left.name.substr(1);
```

Comment:
With the patch in `jsep` is this renaming required?

Reply:
Yes, it is required because the variables are inserted, also, on the style root namespace to ease the implementation of interactivity.